tests/keras/test_callbacks.py

```diff
@@ -381,7 +381,8 @@ def test_TensorBoard_convnet():
                   optimizer='rmsprop',
                   metrics=['accuracy'])
     tsb = callbacks.TensorBoard(log_dir=filepath, histogram_freq=1,
-                                write_images=True, write_grads=True)
+                                write_images=True, write_grads=True,
+                                batch_size=200)
```

Comment:
Please use a smaller batch size in order to fully test the iteration code (here is only a single batch).